packages/api/cms-api/src/action-logs/action-logs.decorator.ts

```diff
@@ -0,0 +1,13 @@
+import type { Permission } from "../user-permissions/user-permissions.types";
+
+export interface ActionLogMetadata {
+    requiredPermission: Array<Permission>;
```

Comment:
`@CrudGenerator()` supports `Permission | Permission[]`, should we support single values here as well?

Reply:
I think there is no need here

Reply:
Let's support both to be consistent.

Reply:
refactored with b2aff28be553fb870e1d7c06fdbbf986948ec36c